web_src/js/components/RepoActionView.vue

```diff
@@ -425,6 +431,23 @@ const sfc = {
       } else {
         actionBodyEl.append(fullScreenEl);
       }
+    },
+    async hashChangeListener() {
+      const selectedLogStep = window.location.hash;
+      const [_, step, _line] = selectedLogStep.split('-');
```

Comment:
```suggestion
      if (!selectedLogStep) return;
      const [_, step, _line] = selectedLogStep.split('-');
```

In case there is no hash.